activities/Stopwatch.activity/css/activity.css

```diff
@@ -144,4 +144,4 @@ button.remove {
 
 .buttons-group button.clear-marks-button {
   background-image: url(../icons/edit-clear.svg);
-}
```

Comment:
No need to make a change where not required.

Reply:
There was only a removal of new line character. Fixed it.